### ISSUE TITLE
JSONDecodeError

### ISSUE THREAD

**Issue by harshraj172** [THUDM/AgentBench#26]
Whenever I run the eval for some models (mostly models hosted via fastchat) I see the below error for some iterations or examples.
 
```
Warning: Exception raised during inference.
Expecting value: line 1 column 1 (char 0)
Traceback (most recent call last):
  File "/home/harsh777111raj/AgentBench/src/agent.py", line 83, in _func
    result = inference_function(messages)
  File "/home/harsh777111raj/AgentBench/src/agents/fastchat_client.py", line 123, in inference
    text = json.loads(line)["text"]
  File "/opt/conda/lib/python3.10/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "/opt/conda/lib/python3.10/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/opt/conda/lib/python3.10/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
Warning: Exception raised during inference.
```

Can you pls tell me the possible reasons for this?

**Comment by zhc7** [comment]
I think this is a fast chat issue. Are you sure you have deployed your sever correctly?

**Comment by Longin-Yu** [comment]
This often happens when an HTTP request encounters an error. If possible, please review the request and response. We've also released a new version, v0.2, and we welcome you to give it a try!

**Comment by Xiao9905** [comment]
@harshraj172 Hi, thanks for your interest in AgentBench. Do you need our further help on the problem? Please feel free to reopen this issue if you need help.
